lblod/frontend-form-builder
# SFB-160: Begrotingstabel

## ID
 [SFB-160](https://binnenland.atlassian.net/browse/SFB-160)

 ## Description

As a user, I want to be able to configure a reusable begrotingstabel, so that it doesn’t need to be custom-developed for every subsidy application.

 ## Type of change


 ## How to test

1. Copy past this code in the code editor of the formbuilder
2. Add some new rows with values to the table
3. See if the total is correct


 ## Links to other PR's

 - /

## Attachments
![image](https://github.com/lblod/frontend-form-builder/assets/36266973/d8510c28-4cd5-43da-affa-6e2face0afb9)
